CMake/FindMbedTLS.cmake

```diff
@@ -0,0 +1,14 @@
+find_path(MBEDTLS_INCLUDE_DIRS mbedtls/ssl.h)
+
+find_library(MBEDTLS_LIBRARY mbedtls)
+find_library(MBEDX509_LIBRARY mbedx509)
+find_library(MBEDCRYPTO_LIBRARY mbedcrypto)
+
+set(MBEDTLS_INCLUDE_DIRS ${MBEDTLS_INCLUDE_DIR})
+set(MBEDTLS_LIBRARIES ${MBEDTLS_LIBRARY} ${MBEDX509_LIBRARY} ${MBEDCRYPTO_LIBRARY})
```

Comment:
Probably a good idea to quote each of these variables (in case it contains spaces). E.g. `"${MBEDTLS_LIBRARY}" "${MBEDX509_LIBRARY}" ...)` (same for previous include_dirs line)